adapters/tappx/tappx.go

```diff
@@ -132,7 +132,7 @@ func (a *TappxAdapter) buildEndpointURL(params *openrtb_ext.ExtImpTappx, test in
 	}
 
 	tappxHost := "tappx.com"
-	isNewEndpoint, err := regexp.Match(`^(zz|vz)[0-9]{3,}([a-z]{2}|test)$`, []byte(params.Endpoint))
+	isNewEndpoint, err := regexp.Match(`^(zz|vz)[0-9]{3,}([a-z]{2,3}|test)$`, []byte(params.Endpoint))
```

Comment:
Hello,
We think that It is not necessary to have a test case for this, because it is a small change to accept 2 or 3 characters in the regex.
Thanks

Reply:
Hello,
Any new about the review?
The change is not really important to add some test case.
Thanks